lib/bundling-job.js

```diff
@@ -8,12 +8,12 @@ async function runBundlingJob (job, archiveName = 'archive.zip') {
   const files = await findCollectionFileMembers(job.used);
   const archivingParams = files.map((f) => {
     return {
-      path: f.physicalUri.replace('share://', '/share/'),
+      path: f.physicalUri.replace('archive://', '/archive/'),
       name: sanitize(f.name, { replacement: '_' })
     };
   });
   const archiveFileMetadata = await createArchive(archiveName, archivingParams);
-  const muFile = await createFile(archiveFileMetadata, archiveFileMetadata.path.replace(/^\/share\//, 'share://'), files[0].graph);
+  const muFile = await createFile(archiveFileMetadata, archiveFileMetadata.path.replace(/^\/archive\//, 'archive://'), files[0].graph);
```

Comment:
This won't work. The [file-service](https://github.com/mu-semtech/file-service#description) doesn't "understand" `archive://` URIs. It always expects the physical file URI (i.e. the URI that encodes the path on disk) to start with `share://`, because it internally uses that convention to translate the URI to a path.
Right now the generated archives are not downloadable by the added `archive-file` service.

Physical files in a semantic.works stack (i.e. the framework we use for Kaleidos) always have to use `share://`.